diskfs_darwin.go

```diff
@@ -0,0 +1,34 @@
+package diskfs
+
+import (
+	"fmt"
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+// this constants should be part of "golang.org/x/sys/unix", but aren't, yet
+const (
+	DKIOCGETBLOCKSIZE         = 0x40046418
+	DKIOCGETPHYSICALBLOCKSIZE = 0x4004644D
```

Comment:
Are these correct? The values are different than the replaced constants, and I am pretty sure (not definitely) that those values were correct for Darwin.

Reply:
they were not. the unix values work on darwin if we're using an image file instead of a block but doesn't work for a `rdisk` block. The values I'm posting works for both cases.

Reply:
👍